### PR TITLE
Use drone-buildx as library in other steps

### PR DESCRIPTION
**Pull request by rutvijmehta-harness**

### CODE REVIEW

File: cmd/drone-gcr/main.go [hunk 1]
```diff
@@ -57,13 +55,7 @@ func main() {
...
 
 	// invoke the base docker plugin binary
-	cmd := exec.Command(docker.GetDroneDockerExecCmd())
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
-	err = cmd.Run()
-	if err != nil {
-		logrus.Fatal(err)
-	}
+	docker.Init()
```
Comment (shubham149) [inline cmd/drone-gcr/main.go:58]: Use Run instead of Init

File: app.go [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+	"github.com/joho/godotenv"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var (
+	version = "unknown"
+)
+
+func Init() {
```
Comment (shubham149) [inline app.go:15]: Use Run instead of init

File: cmd/drone-gcr/main.go (hunk 1 above)
Comment (rutvijmehta-harness) [reply cmd/drone-gcr/main.go]: Done

File: app.go (hunk 2 above)
Comment (rutvijmehta-harness) [reply app.go]: Done

**Review by shubham149 — APPROVED** [review]

**Pull request merged**